requirements_test.txt

```diff
@@ -1,7 +1,9 @@
+types-python-dateutil
+types-mock
 nose
 mock
-black
+snapshottest
+black==21.12b0
```

Comment:
Is there a need to pin this to a beta version? I'd prefer to pin to a standard release version, or no version if possible

Reply:
leaving it at un pinned installed different versions leading to linting inconsistencies. 
Unfortunately **black** is still all beta releases. https://pypi.org/project/black/#history 

Maybe I can pin it to `>=21` ?

Reply:
I see why we would not want to pin a specific version, as it might get yanked. :)

Reply:
Wow I had no idea that all the releases were pre-release. That is an...interesting decision. This is fine, then.